src/interface.rs

```diff
@@ -63,6 +63,25 @@ pub struct Firewall {
 pub struct Link {
     pub master: Option<String>,
     pub mtu: Option<u32>,
+    pub port: Option<LinkPort>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct LinkPort {
+    #[serde(rename = "@type")]
+    pub port_type: LinkPortType,
+    pub priority: Option<u32>,
+    pub path_cost: Option<u32>,
+    pub queue_id: Option<u32>,
+}
+
+#[derive(Debug, PartialEq, SerializeDisplay, DeserializeFromStr, EnumString, Display)]
+#[strum(serialize_all = "lowercase")]
+pub enum LinkPortType {
+    Bridge,
+    Bond,
```

Comment:
The port config of Bond contains the `queue_id`. But agama doesn't have a PortConfig for bonding till now (see: https://github.com/agama-project/agama/blob/master/rust/agama-server/src/network/model.rs#L717) and also there is no `queue_id` parameter in `ifcfg` (see:https://manpages.opensuse.org/Tumbleweed/wicked/ifcfg-bonding.5.en.html).

Reply:
So wouldn't the correct way be to leave it out, thus warning the user that it isn't migrated?

Reply:
yes, you are right. Going to change that.